internal/npm_install/npm_umd_bundle.bzl

```diff
@@ -65,6 +66,9 @@ NPM_UMD_ATTRS = {
         mandatory = True,
         allow_single_file = True,
     ),
+    "excluded": attr.string_list(
+        doc = """List of excluded packages that should not be bundled""",
```

Comment:
needs an example or point to some more doc about why I might need to exclude a package.
Also probably good to point to the CLI option `-u` where we end up plugging this in, since that's authoritative on what are the semantics

Reply:
good point. added more details here